benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -5,7 +5,12 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryConfidentialityRequired", "FieldShowCondition": "appeal.appellant.confidentialityRequired!=\"\"", "TabFieldDisplayOrder": 32},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "isFqpmRequired", "FieldShowCondition": "isFqpmRequired!=\"\"", "TabFieldDisplayOrder": 39},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "isFqpmRequired", "TabFieldDisplayOrder": 38, "FieldShowCondition": "isFqpmRequired!=\"\""},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseName", "TabFieldDisplayOrder": 39},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseNameHmctsInternal", "TabFieldDisplayOrder": 39},
```

Comment:
I don't think we need to display all 3 of these names here, as they will always be the same. Maybe just the `caseNameHmctsInternal` one?